rllab/dynamics_randomization/variation.py

```diff
@@ -0,0 +1,204 @@
+from enum import Enum
+
+
+class VariationMethod(Enum):
+    """
+    The random coefficient is applied according to these methods.
+    """
+    """ The randomization is the product of the coefficient and the dynamic parameter """
+    COEFFICIENT = 1
+    """ The randomization is equal to the coefficient """
+    ABSOLUTE = 2
+
+
+class VariationDistribution(Enum):
```

Comment:
You can just call this `Distribution` since its use is clear inside the package.

Reply:
Done